Documentation/Features/PoA/PoA_idleFedMembersKicker.md

```diff
@@ -0,0 +1,14 @@
+### Idle Federation Members Kicker
+
+This feature kicks federation members in case they were not active for a prolonged period of time. 
+
+To enable it you need to configure the network appropriately:
+
+First of all voting feature should be enabled because kicking is done via voting process among all federation members, kicking a member requires a majority of votes in favor of kicking a fed member. To enable voting you need to set `PoAConsensusOptions.VotingEnabled` to `true`.
```

Comment:
```suggestion
First of all voting the feature should be enabled because kicking is done via a voting process amongst all federation members. Kicking a member requires a majority of votes in favor of kicking a member. To enable voting you need to set `PoAConsensusOptions.VotingEnabled` to `true`.
```